diff --git a/Code/Mantid/MantidQt/CustomInterfaces/src/SANSDiagnostics.cpp b/Code/Mantid/MantidQt/CustomInterfaces/src/SANSDiagnostics.cpp
index 22a221e..b3b9b06 100644
--- a/Code/Mantid/MantidQt/CustomInterfaces/src/SANSDiagnostics.cpp
+++ b/Code/Mantid/MantidQt/CustomInterfaces/src/SANSDiagnostics.cpp
@@ -453,7 +453,7 @@ namespace MantidQt
           // if detector id inside the range
           if (detID >= rectDet->getMinimumDetectorId() && detID <= rectDet->getMaximumDetectorId())
           {
-            aux = spec->getSpectrumNo();
+            aux = spec->getSpectrumNo() -1; // to transfor spectrum number into spectrum index
             if (aux > max_spec_index)
               max_spec_index = aux;
             if (aux < min_spec_index)
