diff --git a/Code/Mantid/Framework/DataHandling/src/LoadEventNexus.cpp b/Code/Mantid/Framework/DataHandling/src/LoadEventNexus.cpp
index 08d6ccf..ef0a3b5 100644
--- a/Code/Mantid/Framework/DataHandling/src/LoadEventNexus.cpp
+++ b/Code/Mantid/Framework/DataHandling/src/LoadEventNexus.cpp
@@ -1519,7 +1519,8 @@ void LoadEventNexus::loadEvents(API::Progress * const prog, const bool monitors)
 
   //----------------- Pad Empty Pixels -------------------------------
   // Create the required spectra mapping so that the workspace knows what to pad to
-  deleteBanks(WS, bankNames);
+  if (WS->getInstrument()->getName().compare("TOPAZ") == 0)
+    deleteBanks(WS, bankNames);
   createSpectraMapping(m_filename, WS, monitors, onebank);
   WS->padSpectra();
 
