diff --git a/Code/Mantid/MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/SANSRunWindow.ui b/Code/Mantid/MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/SANSRunWindow.ui
index 6a52c81..c559764 100644
--- a/Code/Mantid/MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/SANSRunWindow.ui
+++ b/Code/Mantid/MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/SANSRunWindow.ui
@@ -4122,6 +4122,8 @@ p, li { white-space: pre-wrap; }
   </customwidget>
  </customwidgets>
  <tabstops>
+  <tabstop>userfile_edit</tabstop>
+  <tabstop>userfileBtn</tabstop>
   <tabstop>rad_min</tabstop>
   <tabstop>wav_dw</tabstop>
   <tabstop>wav_dw_opt</tabstop>
@@ -4131,13 +4133,16 @@ p, li { white-space: pre-wrap; }
   <tabstop>direct_file</tabstop>
   <tabstop>front_direct_file</tabstop>
   <tabstop>mask_table</tabstop>
-  <tabstop>userfile_edit</tabstop>
-  <tabstop>userfileBtn</tabstop>
   <tabstop>data_dirBtn</tabstop>
   <tabstop>single_mode_btn</tabstop>
   <tabstop>batch_mode_btn</tabstop>
   <tabstop>allowPeriods_ck</tabstop>
   <tabstop>scatterSample</tabstop>
+  <tabstop>scatCan</tabstop>
+  <tabstop>transmis</tabstop>
+  <tabstop>transCan</tabstop>
+  <tabstop>direct</tabstop>
+  <tabstop>dirCan</tabstop>
   <tabstop>rad_max</tabstop>
   <tabstop>csv_filename</tabstop>
   <tabstop>csv_browse_btn</tabstop>
